CONTRIBUTING.md

```diff
@@ -29,27 +124,31 @@ The documentation is translated into these languages:
+To translate expressjs.com into a new language, follow these steps:
 
+1. Clone the [`expressjs.com`](https://github.com/expressjs/expressjs.com) repository.
+2. Create a directory for the language of your choice using its [ISO 639-1 code](https://www.loc.gov/standards/iso639-2/php/code_list.php) as its name.
+3. Copy `index.md`, `api.md`, `starter/`, `guide/`, `advanced/`, `resources/`, `4x/`, and `3x/`, to the language directory.
+4. Remove the link to 2.x docs from the "API Reference" menu.
+5. Update the `lang` variable in the copied markdown files.
+6. Update the `title` variable in the copied markdown files.
+7. Create the header, footer, notice, and announcement file for the language in the `_includes/` directory, in the respective directories, and make necessary edits to the contents.
+8. Create the announcement file for the language in the `_includes/` directory.
+9. Make sure to append `/{{ page.lang }}` to all the links within the site.
+10. Update the [CONTRIBUTING.md](https://github.com/expressjs/expressjs.com/blob/gh-pages/CONTRIBUTING.md#contributing-translations)  and the `.github/workflows/translation.yml` files with the new language. 
 
-### Adding new translations
+### Adding Page and Section Translations
 
-To contribute a translation into another language, following the procedure below.
+Many site translations are still missing pages. To find which ones we need help with, you can [filter for merged PRs](https://github.com/expressjs/expressjs.com/pulls?q=is%3Apr+is%3Aclosed+label%3Arequires-translation-es) that include the tag for your language, such as `requires-translation-es` for requires Spanish translation.   
 
-Follow these steps:
+If you contribute a page or section translation, please reference the original PR. This helps the person merging your translation to remove the tag from the original PR.
 
-0. Clone the [`expressjs.com` repository](https://github.com/expressjs/expressjs.com)
-1. Create a directory for the language of your choice using its [ISO 639-1 code](http://www.loc.gov/standards/iso639-2/php/code_list.php) as its name.
-2. Copy `index.md`, `api.md`, `starter/`, `guide/`, `advanced/`, `resources/`, `4x/`, and `3x/`, to the language directory.
-3. Remove the link to 2.x docs from the "API Reference" menu.
-4. Update the `lang` variable in the copied markdown files.
-5. Update the `title` variable in the copied markdown files.
-6. Create the header, footer, notice, and announcement file for the language in the `_includes/` directory, in the respective directories, and make necessary edits to the contents.
-7. Create the announcement file for the language in the `_includes/` directory.
-9. Make sure to append `/{{ page.lang }}` to all the links within the site.
-10. Update the `CONTRIBUTING.md` and the `.github/workflows/translation.yml` files with the new language
+## We're the Express JS Framework's Documentation Team - Not the Express JS Framework
```

Comment:
This is out of place at the very end of the document.  It doesn't do any good here.  It should be combined with the existing info at the very beginning, where it will do some good... Most of the info is already there already anyway so I think this could just be removed.

Reply:
Awesome feedback @crandmck! Will make the changes.